circle.yml

```diff
@@ -0,0 +1,16 @@
+dependencies:
+  override:
+    - yarn
+  pre:
+    - rm -rf ~/.yarn
+    - npm install -g yarn
```

Comment:
Your example seems also to have `yarn install` in it (but followed by `npm rebuild`)? Is it another way to skin the same cat? Don't know enough about `npm` and `yarn` to comment.